src/rpc/client.cpp

```diff
@@ -142,8 +142,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "keypoolrefill", 0, "newsize" },
     { "getrawmempool", 0, "verbose" },
     { "estimatefee", 0, "nblocks" },
-    { "estimatesmartfee", 0, "nblocks" },
-    { "estimaterawfee", 0, "nblocks" },
+    { "estimatesmartfee", 0, "conf_target" },
+    { "estimaterawfee", 0, "conf_target" },
     { "estimaterawfee", 1, "threshold" },
     { "prioritisetransaction", 1, "fee_delta" },
     { "setban", 2, "bantime" },
```

Comment:
missing changes in mining.cpp 

`
     { "util",               "estimatesmartfee",       &estimatesmartfee,       {"nblocks", "estimate_mode"} }, 
` ->
`
     { "util",               "estimatesmartfee",       &estimatesmartfee,       {"conf_target", "estimate_mode"} }, 
`

Reply:
That's because this was already done in #3096 when we backported 10707.